docs/1.essentials/4.steps_leave.md

```diff
@@ -0,0 +1,27 @@
+## Steps to discontinue a FIWARE Lab node
+
+In order to discontinue the operation of a running FIWARE Lab node there
+are several activities and obligations to be respected. The most
+important, which is the base of the FIWARE philosophy, is taking care of
+the migration of the resources toward a persistent node, in order to
```

Comment:
here it says "migration of resources", and in the list of tasks "migration of users". Maybe it could be clarified which things should actually be migrated: only the user accounts, the resources they had, the services they had? etc.